mobilecoind/strategies/drain-accounts.py

```diff
@@ -141,20 +141,25 @@ def filename_key(filename):
         for b58 in dest_b58addresses
     ]
 
-    # Go through each account and have all their friends transact to them
-    for i, (src_account, dest) in enumerate(zip(source_accounts, dest_addresses)):
-        # If we have already done max_accounts many accounts, then stop
-        if i == args.max_accounts:
-            break;
+    # Go through each destination account and pay it once from a source account,
+    # going round-robin through the source accounts (only first args.num_src_accounts source accounts)
+    n = len(dest_addresses)
+    m = len(source_accounts) if args.num_src_accounts == -1 else args.num_src_accounts
```

Comment:
`n/m` are confusing names, can you switch to `num_dst_accounts`/`num_src_accounts`?